test/python/WMComponent_t/DBS3Buffer_t/DBSUpload_t.py

```diff
@@ -638,6 +638,15 @@ def testPassiveExceptions(self):
                            'OpenSSL SSL_connect: SSL_ERROR_SYSCALL']:
             self.assertIs(isPassiveError(MyTestException(errMessage)), True)
 
+    def testParseDBSException(self):
```

Comment:
Please add tests mimic'ing a FE timeout (Proxy Error), where we might not have a `reason` key.
It might be useful to add yet another test where exception is a python object (not a json).